### PR TITLE
change import path to use vanity URL sigs.k8s.io

### PR DESCRIPTION
**Pull request by mengqiy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ to simplify building and publishing Kubernetes APIs from scratch.
 
 ### Installalation 
 
-It is strongly recommended that you use a released version. Release binaries are available on the [releases](https://github.com/kubernetes-sigs/kubebuilder/releases) page.
+It is strongly recommended that you use a released version. Release binaries are available on the [releases](https://sigs.k8s.io/kubebuilder/releases) page.
```
Comment (Liujingfang1) [inline README.md]: No need to change this. This link is a 404
`https://sigs.k8s.io/kubebuilder/releases`

File: README.md [hunk 2]
```diff
@@ -27,10 +27,10 @@ Check out the Kubebuilder [book](http://book.kubebuilder.io).
 ## Resources
 
 - GitBook: [book.kubebuilder.com](http://book.kubebuilder.io)
-- GitHub Repo: [kubernetes-sigs/kubebuilder](https://github.com/kubernetes-sigs/kubebuilder)
+- GitHub Repo: [kubernetes-sigs/kubebuilder](https://sigs.k8s.io/kubebuilder)
 - Slack channel: [#kubebuilder](http://slack.k8s.io/#kubebuilder)
 - Google Group: [kubebuilder@googlegroups.com](https://groups.google.com/forum/#!forum/kubebuilder)
-- Planned Work: [Sprint Dashboard](https://github.com/kubernetes-sigs/kubebuilder/projects/1)
+- Planned Work: [Sprint Dashboard](https://sigs.k8s.io/kubebuilder/projects/1)
```
Comment (Liujingfang1) [inline README.md]: This page is also not found

File: README.md [hunk 3]
```diff
@@ -74,7 +74,7 @@ Provide clean library abstractions with clear and well exampled godocs.
 ## Troubleshooting
 
 - ### Bugs and Feature Requests:
-  If you have what looks like a bug, or you would like to make a feature request, please use the [Github issue tracking system.](https://github.com/kubernetes-sigs/kubebuilder/issues)
+  If you have what looks like a bug, or you would like to make a feature request, please use the [Github issue tracking system.](https://sigs.k8s.io/kubebuilder/issues)
```
Comment (Liujingfang1) [inline README.md]: And this one as well.

File: build/build_kubebuilder.sh [hunk 4]
```diff
@@ -22,10 +22,10 @@ set -x
 # - Use /go as the default GOPATH because this is what the image uses
 # - Link our current directory (containing the source code) to the package location in the GOPATH
 
-OWNER="kubernetes-sigs"
+OWNER="sigs.kubernetes.io"
```
Comment (Liujingfang1) [inline build/build_kubebuilder.sh]: I think this one should be "sigs.k8s.io"

**Comment by Liujingfang1** [comment]
You may need to trigger a release to make sure it works with the updated import path.

**Comment by mengqiy** [comment]
> You may need to trigger a release to make sure it works with the updated import path.

How can we trigger a release w/o merging the PR? Any idea?

**Comment by Liujingfang1** [comment]
> > You may need to trigger a release to make sure it works with the updated import path.

> How can we trigger a release w/o merging the PR? Any idea?

You can try this 
```cloud-build-local \
   --config=build/cloudbuild_local.yaml \
   --dryrun=false --write-workspace=/tmp/w .
```

If it doesn't work, we can merge this PR. Then push a temporary tag to trigger it.

**Comment by mengqiy** [comment]
Cloud build is failing due to the deletion of `pkg` dir yesterday.
I will fix that before merging this PR.

**Comment by mengqiy** [comment]
Fixed cloud build in https://github.com/kubernetes-sigs/kubebuilder/pull/429
/hold cancel

**Pull request merged**